### ISSUE TITLE
Package not showing in package list

### ISSUE THREAD

**Issue by rosshemsley** [wbond/package_control#657]
It seems my package "iOpener" is not being displayed in the install results for some users.

I had the same problem originally, and attributed it to a problem with my package control configuration file: I reset it by deftly copying the version from the PC repo into my own, and then it mysteriously it started working again.

Here is the message I received from the user concerned:

```
Hi,
I was willing to try your plugin, but I can't find it in Package Control. I searched it with    
"iOpener", "opener", "i open", etc… but it never shown up.

I'm on Linux, I tried with both ST 2 and 3 with an up-to-date version of PC, and several       
times over the past week.
```

**Comment by iRonin** [comment]
Today I switched to ST3 and I run into the same issue…
I was able to install a few `SublimeLinter-*` packages but they wouldn't work; I realized I didn't have the `SublimeLinter` package itself so I tried to install it but it wouldn't show up. I removed the other linter packages but still no juice and now they also won't show up.
I'm using `  Package Control Version: 3.0.0`.
I checked https://packagecontrol.io/channel_v3.json and the packages are there but they are not listed for me. 
If I do advanced install I get `Package Control: The package "SublimeLinter" is either not available on this platform or for this version of Sublime Text`…
